integration-tests/utils/integration-utils.js

```diff
@@ -191,10 +191,29 @@ const dismissNotifications = async page => {
   return !!notificationCloseButtons.length && delay(1000) // delayed for alerts to animate off
 }
 
+const loadSignInPageAndWaitForReady = (page, url, timeout = 30 * 1000) => {
```

Comment:
Nit: The following reads a bit more naturally to me.

```suggestion
const loadSignInPageAndWaitUntilReady = (page, url, timeout = 30 * 1000) => {
```